src/payment/strategies/cybersource/cardinal.ts

```diff
@@ -50,7 +52,7 @@ export interface CardinalOrderData {
     currencyCode: string;
     id: string;
     amount: number;
-    paymentData: CreditCardInstrument;
+    paymentData?: CreditCardInstrument;
```

Comment:
Same than the type above, this is a responsibility of the `cardinal-client` not the `cardinal-sdk`. Could we move this whole type (`CardinalOrderData`) to the `cardinal-client` file?